e2e/local-artifact-mirror_test.go

```diff
@@ -59,5 +60,30 @@ func TestLocalArtifactMirror(t *testing.T) {
 		t.Fatalf("we should not be able to fetch paths with ../")
 	}
 
+	t.Logf("testing local artifact mirror restart after materialize")
+	commands = [][]string{
+		{"embedded-cluster", "materialize"},
+		{"sleep", "20"},
+	}
+	if err := RunCommandsOnNode(t, tc, 0, commands); err != nil {
+		t.Fatalf("fail materialize embedded cluster binaries: %v", err)
+	}
+
+	command = []string{"journalctl", "-u", "local-artifact-mirror"}
+	if _, _, err := RunCommandOnNode(t, tc, 0, command); err != nil {
+		t.Fatalf("fail to get journalctl logs: %v", err)
+	}
+
+	expected := []string{
+		"Binary changed, sending signal to stop",
+		"Scheduled restart job, restart counter is at",
+	}
```

Comment:
Are there logs that are created upon startup? Could we somehow look for that showing up after the "binary changed" message?

Reply:
The `Scheduled restart job, restart counter is at` is logged upon restart.